servers/rendering/renderer_rd/shader_rd.cpp

```diff
@@ -234,9 +234,11 @@ void ShaderRD::_build_variant_code(StringBuilder &builder, uint32_t p_variant, c
 					builder.append(String("#define ") + String(E.key) + "_CODE_USED\n");
 				}
 #if (defined(MACOS_ENABLED) || defined(APPLE_EMBEDDED_ENABLED))
+#if defined(__x86_64) || defined(__x86_64__)
```

Comment:
I believe `__x86_64__` should always work, no? So `__x86_64` is likely redundant and not necessary.